### PR TITLE
feat: :sparkles: updated detail page to match the figma demo

### PR DESCRIPTION
**Pull request by snenenenenenene**
DETAIL PAGE
- added session display to the bottom of the detail page
- added an indicator on whether or not an agenda item has been decided
- added styling that looks like the figma document
- added a resolution ember data model to look if agenda item has been decided
- removed contact link on section to the right of the page

### CODE REVIEW

File: app/controllers/home.ts [hunk 1]
```diff
@@ -4,11 +4,39 @@ import { action } from "@ember/object";
...
+  @tracked errorMsg = "";
+
+  @action
+  async loadMore() {
+    if (this.model && !this.isLoadingMore) {
+      this.isLoadingMore = true;
+      const nextPage = this.model.currentPage + 1;
+      const agendaItems = await this.store.query(
+        "agenda-item",
+        this.model.getQuery({ page: nextPage })
```
Comment (aliokan) [inline app/controllers/home.ts:27]: Shouldn't we add the filters here?

File: tests/unit/utils/get-formatted-date-range-test.js [hunk 2]
```diff
@@ -1,23 +0,0 @@
-import { module, test } from 'qunit';
-import { getFormattedDateRange } from 'frontend-burgernabije-besluitendatabank/utils/get-formatted-date-range';
-
-module('Unit | Utility | get-formatted-date-range', function () {
```
Comment (aliokan) [inline tests/unit/utils/get-formatted-date-range-test.js:4]: Why did you remove these tests?

File: tests/unit/utils/get-formatted-date-test.js [hunk 3]
```diff
@@ -1,17 +0,0 @@
-import { module, test } from 'qunit';
-import { getFormattedDate } from 'frontend-burgernabije-besluitendatabank/utils/get-formatted-date';
-
-module('Unit | Utility | get-formatted-date', function () {
```
Comment (aliokan) [inline tests/unit/utils/get-formatted-date-test.js:4]: Same here

**Review by aliokan — CHANGES_REQUESTED** [review]
I add some comments

File: app/controllers/home.ts (hunk 1 above)
Comment (snenenenenenene) [reply app/controllers/home.ts:28]: I thought filtering should be agnostic to paginaton so I added it all to the route

File: tests/unit/utils/get-formatted-date-range-test.js (hunk 2 above)
Comment (snenenenenenene) [reply tests/unit/utils/get-formatted-date-range-test.js:4]: I heard echoes of jodi telling us testing is for the testing team

File: app/templates/home.hbs [hunk 4]
```diff
@@ -1,50 +1,51 @@
...
+        @placeholder={{"Alle gemeentes"}}
+        @noMatchesMessage={{"Geen gemeente gevonden"}}
+        @searchMessage="Aan het zoeken..."
+        @searchEnabled={{true}}
+        class="au-u-margin-right-small"
+        as |singleSelect|
+      >
+        {{singleSelect.label}}
+      </PowerSelect>
+      <AuButton
```
Comment (aliokan) [inline app/templates/home.hbs:41]: The button has to be disabled when municipality is empty, or at least do not add empty filter in url `gemeentes=`. 
image

**Review by aliokan — CHANGES_REQUESTED** [review]

File: app/controllers/home.ts (hunk 1 above)
Comment (aliokan) [reply app/controllers/home.ts:28]: I agree with you pagination doesn't need to know about filter, but with your current implementation the loading nextPage doesn't apply filters. You can inject filters in `this.model.getQuery` from the route.

File: app/routes/home.ts [hunk 5]
```diff
@@ -49,14 +127,59 @@ export default class HomeRoute extends Route {
...
+  async model(params: any, transition: Transition<unknown>) {
+    // const model: any = this.modelFor("home");
+    // if (
+    //   model?.agendaItems?.toArray().length > 0 &&
+    // ) {
+    //   return model;
+    // }
+
+    const currentPage = 0;
+    const agendaItems = await this.store.query(
```
Comment (aliokan) [inline app/routes/home.ts:156]: We do not need to load agenda items to display the new Home page. 
image

File: app/controllers/home.ts (hunk 1 above)
Comment (snenenenenenene) [reply app/controllers/home.ts]: Fixed!

File: app/templates/home.hbs [hunk 6]
```diff
@@ -12,7 +12,7 @@
   style="display: flex; justify-content: center; align-items: center;"
 >
 
-  <AuFormRow class="au-u-2-5">
```
Comment (aliokan) [inline app/templates/home.hbs:15]: Is `au-u-2-5` not working?

File: tests/integration/components/vote-table-entry-test.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
```
Comment (aliokan) [inline tests/integration/components/vote-table-entry-test.js:1]: that test can be removed ;)

**Review by aliokan — CHANGES_REQUESTED** [review]
left 2 small comments
